### PR TITLE
add hugegraph-computer quick start

### PR DESCRIPTION
**Pull request by coderzc**

### CODE REVIEW

**Comment by imbajin** [comment]
hung it, wait `website` branch changes to master

File: quickstart/hugegraph-computer.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+## HugeGraph-Computer Quick Start
```
Comment (javeme) [inline quickstart/hugegraph-computer.md:1]: can we also add a Chinese version

File: quickstart/hugegraph-computer.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+### 1 Overviews
+
+HugeGraph-Computer is a large-scale graph computing system, basic on disk/memory & integrate with graph database HugeGraph.
```
Comment (javeme) [inline quickstart/hugegraph-computer.md:5]: it is based on disk/memory and integrates with graph database HugeGraph.

File: quickstart/hugegraph-computer.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
```
Comment (javeme) [inline quickstart/hugegraph-computer.md:6]: can add some description for what algorithms we provide.

We provide a list of builtin graph algorithms:
| Builtin Algorithm Name | Description |
| page_rank | Calculate the PageRank value for each vertex in the graph |
| xx | xx |

**Pull request closed without merging**